### ISSUE TITLE
Fields are not updated in object_set/update when mixing plain and free columns

### ISSUE THREAD

**Issue by jiho** [ecotaxa/ecotaxa_dev#556]
```
{"target_ids":[24473014],"updates":[{"ucol":"longitude","uval":0},{"ucol":"latitude","uval":0}]} 
```
updates latitude and longitude, two plain columns

```
{"target_ids":[24473014],"updates":[{"ucol":"area","uval":1},{"ucol":"mean","uval":1}]} 
```
updates area (n03) and mean (n04), two free columns

```
{"target_ids":[24473014],"updates":[{"ucol":"area","uval":10},{"ucol":"latitude","uval":10}]} 
```
updates area (n03) but not latitude = only the free column

```
{"target_ids":[24473014],"updates":[{"ucol":"area","uval":9},{"ucol":"latitude","uval":9},{"ucol":"mean","uval":9}]} 
```
updates area (n03) and mean (n04) but not latitude = only the free columns

```
{"target_ids":[24473014],"updates":[{"ucol":"latitude","uval":5},{"ucol":"area","uval":5}]} 
{"target_ids":[24473014],"updates":[{"ucol":"latitude","uval":4},{"ucol":"area","uval":4},{"ucol":"longitude","uval":4}]} 
```
updates latitude (and longitude) but not area (n03) = only the plain column(s).

So it seems that, when mixing plain and free columns, the code only update one kind and the updated kind is whichever comes first in the query.

NB: To restore the original values
```
{"target_ids":[24473014],"updates":[{"ucol":"longitude","uval":7.3157},{"ucol":"latitude","uval":43.685}]} 
{"target_ids":[24473014],"updates":[{"ucol":"area","uval":1028},{"ucol":"mean","uval":224}]}

**Comment by grololo06** [comment]
Indeed. Makes me think that the return value of this primitive (number of affected rows) is not documented and should be.

**Comment by jiho** [comment]
I discovered this quickly with the examples of use (in https://github.com/ecotaxa/ecotaxa_back/blob/master/QA/py/tests/test_update.py). That said, it does not help discover the bug since, even when mixing plain and free fields, the number of affected *rows* is correct. It is the number of affected *values* that is incorrect.

**Comment by grololo06** [comment]
Fixed in 2.5.1.
